tests/utils/common.py

```diff
@@ -86,13 +88,18 @@ def load_from_settings_file(self):
             self.__dict__ = load_dict_from_json_file(custom)
 
     def get_settings_path(self):
-        return os.path.join(get_root_dir(), SETTINGS_FILE)
+        return os.path.join(get_root_dir(), self.settings_file)
 
     def register_pytest_command_line_options(self, parser):
         for key, val in object_dict_items(self):
             parser.addoption("--%s" % key, action="store", default=None)
+        parser.addoption("--settings_file", action="store", default=None)
 
     def load_from_pytest_command_line(self, config):
+        if config.getoption('settings_file'):
+            globals()['SETTINGS_FILE'] = config.getoption('settings_file')
```

Comment:
this construct is useful when we need a one-liner. since we're using globals twice, better use this construct.
```python
global SETTINGS_FILE
SETTINGS_FILE = ...

self.settings_file = SETTINGS_FILE
```

Reply:
addressed